lib/linux_admin/service/sys_v_init_service.rb

```diff
@@ -42,5 +42,14 @@ def restart
 
       self
     end
+
+    def reload
```

Comment:
Should both of these commands live in the service class since they are exact copies?